src/sdk/namespace_cache.js

```diff
@@ -157,18 +168,32 @@ class NamespaceCache {
     /////////////////
 
     async list_objects(params, object_sdk) {
-        // TODO listing from cache only for deevelopment
-        return this.namespace_nb.list_objects(params, object_sdk);
+        const get_from_cache = params.get_from_cache;
+        params = _.omit(params, 'get_from_cache');
+        if (get_from_cache) {
+            return this.namespace_nb.list_objects(params, object_sdk).then(res => {
```

Comment:
Avoid using `promise.then()`, use `await` instead -
```suggestion
            const res = await this.namespace_nb.list_objects(params, object_sdk);
```